src/logging.cpp

```diff
@@ -184,6 +184,47 @@ bool GetLogCategory(BCLog::LogFlags& flag, const std::string& str)
     return false;
 }
 
+const std::unordered_map<BCLog::Level, std::string> LevelStr{
+    {BCLog::Level::None, "none"},
+    {BCLog::Level::Info, "info"},
+    {BCLog::Level::Debug, "debug"},
+    {BCLog::Level::Warning, "warning"},
+    {BCLog::Level::Error, "error"},
+};
+
+const std::unordered_map<BCLog::LogFlags, std::string> LogFlagStr{
```

Comment:
nit: Maybe name this LogCategoryStr, as flags (combined categories) are ignored?

Reply:
Done 👍🏻 . Replaced this unordered map with a helper function and named it `LogCategoryToStr`